### ISSUE TITLE
1.1.0.2 on Windows - Stuck at "Selecting package versions"

### ISSUE THREAD

**Issue by manugill** [meteor/meteor#4708]
Even the spinner is stuck. Ctrl + C doesn't do anything. Although, I can close Node from Task Manager and it returns me back to CMD.

It started happening after I restarted my computer while running Meteor (which I admit is kind of stupid to do). Before that, it was working fine. I've tried to reset my project, remove packages to keep the bare minimum and reinstall Meteor, the problem still persists.

**Comment by Slava** [comment]
Does it happen on any app or just this app? Have you tried running a bare boilerplate app that you get after running `meteor create bla`? If it is the issue with selecting package versions, the list of packages your app uses would be the reproduction here.

**Comment by manugill** [comment]
All apps, I tried installing boilerplates and the todos app and creating a new meteor app as well. They could be installed but all of them were stuck.

I can't reproduce it anymore though, reinstalled Windows last night and it is working fine on a fresh install.

Anyway, here is a list of the packages I had installed at that point time
meteor-platform
msavin:mongol
msavin:jetsetter
accounts-password
manuel:viewmodel
aldeed:autoform
iron:router
aldeed:collection2
fourseven:scss
percolate:velocityjs
reactive-var
juliancwirko:s-alert
juliancwirko:s-alert-scale
alethes:pages
cfs:standard-packages
cfs:filesystem
cfs:autoform
ewall:foundation
